docs/sources/developers/plugins/add-support-for-variables.md

```diff
@@ -51,14 +51,34 @@ For data sources, you need to use the [getTemplateSrv]({{< relref "../../package
 
    ```ts
    async query(options: DataQueryRequest<MyQuery>): Promise<DataQueryResponse> {
-     const query = getTemplateSrv().replace('SELECT * FROM services WHERE id = "$service"'), options.scopedVars);
+     const query = getTemplateSrv().replace('SELECT * FROM services WHERE id = "$service"', options.scopedVars);
 
      const data = makeDbQuery(query);
 
      return { data };
    }
    ```
 
+## Format multiple variables
+
+If the user has selected multiple values, the value of the interpolated variable depends on the [variable format](https://grafana.com/docs/grafana/latest/variables/advanced-variable-format-options/).
```

Comment:
```suggestion
If the user selects multiple values, then the value of the interpolated variable depends on the [variable format](https://grafana.com/docs/grafana/latest/variables/advanced-variable-format-options/).
```

Reply:
Unclear what you mean with "multiple values." Do you mean multiple variable values? A variable with multiple values? Multi-value variables? Selected how? In the dropdown? Please clarify.

Reply:
All of what you said is correct :) Maybe "When a user selects multiple values for variable"?

Reply:
Yes, that makes more sense.